Patches/Vanilla Factions Expanded - Settlers/PawnKindDefs/PawnKinds_Settlers.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Combat Extended</li>
+    </mods>
+
+    <match Class="PatchOperationSequence">
+      <success>Always</success>
```

Comment:
Is the `<success>Always</success>` required here? I tend to prefer not hiding errors when they occur.

Reply:
![image](https://user-images.githubusercontent.com/1188951/79497112-d4c94580-802f-11ea-8782-e8ea4b37d4a9.png)